### PR TITLE
Add .pep8speaks.yml

### PR DESCRIPTION
**Pull request by jp-hoehmann**

### CODE REVIEW

**Comment by NuvandaPV** [comment]
fixes #1959

File: .pep8speaks.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        - E127 # Continuation line over-indented for visual indent.
+        - E128 # Continuation line under-indented for visual indent.
+        - E129 # Visually indented line with same indent as next logical line.
+        - E131 # Continuation line unaligned for hanging indent.
+        - E133 # Closing bracket is missing indentation.
+        - E226 # Missing whitespace around arithmetic operator,
+        - E241 # Multiple spaces after `,`.
+        - E402 # Module level import not at the top of file.
+        - E704 # Multiple statements on one line (def).
+        - E711 # Comparison to None should be `if cond is None:`.
```
Comment (cornelinux) [inline .pep8speaks.yml]: please remove this

File: .pep8speaks.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - E712 # Comparison to True should be `if cond is True:` or `if cond:`.
```
Comment (cornelinux) [inline .pep8speaks.yml]: remove this

File: .pep8speaks.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        - E124 # Closing bracket does not match visual indentation.
+        - E125 # Continuation line with same indent as next logical line.
+        - E126 # Continuation line over-indented for hanging indent.
+        - E127 # Continuation line over-indented for visual indent.
+        - E128 # Continuation line under-indented for visual indent.
+        - E129 # Visually indented line with same indent as next logical line.
+        - E131 # Continuation line unaligned for hanging indent.
+        - E133 # Closing bracket is missing indentation.
+        - E226 # Missing whitespace around arithmetic operator,
+        - E241 # Multiple spaces after `,`.
```
Comment (cornelinux) [inline .pep8speaks.yml]: this is cool. please remove this

File: .pep8speaks.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        - E122 # Continuation line missing indentation or outdented.
+        - E124 # Closing bracket does not match visual indentation.
+        - E125 # Continuation line with same indent as next logical line.
+        - E126 # Continuation line over-indented for hanging indent.
+        - E127 # Continuation line over-indented for visual indent.
+        - E128 # Continuation line under-indented for visual indent.
+        - E129 # Visually indented line with same indent as next logical line.
+        - E131 # Continuation line unaligned for hanging indent.
+        - E133 # Closing bracket is missing indentation.
+        - E226 # Missing whitespace around arithmetic operator,
```
Comment (cornelinux) [inline .pep8speaks.yml]: remove this

File: .pep8speaks.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        - E121 # Continuation line under-indented for hanging indent.
+        - E122 # Continuation line missing indentation or outdented.
+        - E124 # Closing bracket does not match visual indentation.
+        - E125 # Continuation line with same indent as next logical line.
+        - E126 # Continuation line over-indented for hanging indent.
+        - E127 # Continuation line over-indented for visual indent.
+        - E128 # Continuation line under-indented for visual indent.
+        - E129 # Visually indented line with same indent as next logical line.
+        - E131 # Continuation line unaligned for hanging indent.
+        - E133 # Closing bracket is missing indentation.
```
Comment (cornelinux) [inline .pep8speaks.yml]: change this to E123

File: .pep8speaks.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        - E743 # Do not use functions named `l`, `O`, or `I`.
+        - W503 # Line break before binary operator.
+        - W505 # Doc line too long.
+    exclude: []
+    count: False
+    first: False
+    show-pep8: False
+    show-source: False
+    statistics: False
+    hang-closing: False
```
Comment (cornelinux) [inline .pep8speaks.yml]: Set to "True"

**Review by cornelinux — CHANGES_REQUESTED** [review]

**Review by cornelinux — APPROVED** [review]

**Pull request merged**